analyzer/consensus/consensus.go

```diff
@@ -299,25 +301,29 @@ func (m *processor) ProcessBlock(ctx context.Context, uheight uint64) error {
 		return fmt.Errorf("height %d is too large", uheight)
 	}
 	height := int64(uheight)
+	batch := &storage.QueryBatch{}
 
-	// Fetch all data.
-	fetchTimer := m.metrics.BlockFetchLatencies()
-	data, err := fetchAllData(ctx, m.source, m.network, height, m.mode == analyzer.FastSyncMode)
-	fetchTimer.ObserveDuration()
-	if err != nil {
-		if strings.Contains(err.Error(), fmt.Sprintf("%d must be less than or equal to the current blockchain height", height)) {
-			return analyzer.ErrOutOfRange
+	isBlockAbsent := m.chain == common.ChainNameMainnet && height == 8048955 // Block does not exist due to mishap during upgrade to Damask.
```

Comment:
just curious - what do you think of something like 
```suggestion
	if m.chain == common.ChainNameMainnet && height == 8048955 { // Block does not exist due to mishap during upgrade to Damask.
	    return nil
	}
```

Reply:
nvm we still have to update the indexing progress